controller/src/main.cpp

```diff
@@ -111,6 +111,16 @@ static Controller controller;
 static ControllerStatus controller_status;
 static Sensors sensors;
 
+static SensorsProto AsSensorsProto(SensorReadings r) {
+  SensorsProto proto = SensorsProto_init_zero;
+  proto.patient_pressure_cm_h2o = r.patient_pressure.cmH2O();
+  proto.inflow_pressure_diff_cm_h2o = r.inflow_pressure_diff.cmH2O();
+  proto.outflow_pressure_diff_cm_h2o = r.outflow_pressure_diff.cmH2O();
+  proto.flow_ml_per_min = (r.inflow - r.outflow).ml_per_min();
```

Comment:
Per previous comment, would we use corrected flow here? Or is that corrected flow only necessary for TV computation?

Reply:
Really good point; done.